test/account.test.js

```diff
@@ -185,14 +196,17 @@ describe('with deploy contract', () => {
         expect(result).toEqual('hello trex');
 
         const setCallValue = testUtils.generateUniqueString('setCallPrefix');
-        const result2 = await contract.setValue({ value: setCallValue });
+        const result2 = await contract.setValue({ args: { value: setCallValue } });
         expect(result2).toEqual(setCallValue);
         expect(await contract.getValue()).toEqual(setCallValue);
     });
 
     test('make function calls via contract with gas', async() => {
         const setCallValue = testUtils.generateUniqueString('setCallPrefix');
-        const result2 = await contract.setValue({ value: setCallValue }, 1000000 * 1000000);
+        const result2 = await contract.setValue({
+            args: { value: setCallValue },
+            gas: 1000000 * 1000000
```

Comment:
Curious why we are doing a calculation here rather than just passing the actual amount.

Reply:
It's easier to count zeros this way!
It's even more useful when we are talking about time. Like, `24*60*60` is better than `86400`.